examples/elastic-cloud-cluster-integration/details.md

```diff
@@ -1,16 +1,27 @@
 ## Self-managed ROR cluster and remote Elastic Cloud cluster configuration details
 
+This is a detailed description of how to configure two Elasticsearch clusters:
+1. One in Elastic Cloud (managed Elasticsearch from Elastic) containing the bulk of the data 
+2. One self hosted with ReadonlyREST  (for enterprise-level access control and authentication)
+
+The objective is to get the two connected using the transport protocol over SSL, so that we can attach a Kibana (with ROR Enterprise installed) to the cluster #2, and from there query the data in cluster #1 using the [Cross Cluster Search (CCS)](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-cross-cluster-search.html) feature. 
+
+
 ### Two-way SSL configuration
 
 The local, self-managed ROR cluster connects with the remote Elastic Cloud cluster using the Elasticsearch transport interface.
-The transport uses two-way SSL to authorize nodes of clusters. To do that, we need to generate CA certificates of nodes of 
-the local cluster (using the CA certificates of the Elastic cloud cluster). Then we need to use them to add a trusted environment 
-in the Elastic Cloud cluster. CA certificates of the Elastic Cloud cluster nodes can be downloaded from the security settings of 
+The transport uses two-way SSL to authorize nodes of clusters.
+
+To do that, we need to 
+1. Generate CA certificates of nodes of the local cluster (using the CA certificates of the Elastic cloud cluster)
+2. Use them to add a trusted environment in Elastic Cloud console 
+
```

Comment:
3. Configure the internode SSL and remote cluster settings in `elasticsearch.yml`

WDYT?